src/Scripting/CSharpTest/InteractiveSessionTests.cs

```diff
@@ -394,6 +394,18 @@ public void Fields_Visibility()
             Assert.Equal(16, script.ContinueWith<int>("pi").EvaluateAsync().Result);
         }
 
+        [WorkItem(100639)]
+        [Fact]
+        public void ExternDestructor()
+        {
+            var script = CSharpScript.Create(
+@"class C
+{
+    extern ~C();
+}");
+            Assert.Null(script.EvaluateAsync().Result);
```

Comment:
The test would have resulted in an exception back when the bug was logged, but there have been significant changes to emit since then so that code path is no longer hit. But it seemed like an interesting test case.